fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy has a fix for the issue: [Replace `'.jsSelectedMainEmail'` with `".jsSelectedMainEmail"`](https://app.codacy.com/gh/prodcodacytestuser/CheckPrCommentsPublicProjectProd/pullRequest?prid=9341204)
```suggestion
  $(".jsSelectedMainEmail").text(postVariable.email);
```